mls-rs/src/group/proposal.rs

```diff
@@ -232,6 +232,29 @@ impl Debug for ExternalInit {
     }
 }
 
+#[derive(Clone, PartialEq, Eq, MlsSize, MlsEncode, MlsDecode)]
+#[cfg_attr(feature = "arbitrary", derive(arbitrary::Arbitrary))]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg(all(
+    feature = "by_ref_proposal",
+    feature = "custom_proposal",
+    feature = "self_remove_proposal"
+))]
+/// A proposal to remove the current [`Member`](mls_rs_core::group::Member) of a
+/// [`Group`](crate::group::Group).
+pub struct SelfRemoveProposal {}
+
+#[cfg(all(
+    feature = "by_ref_proposal",
+    feature = "custom_proposal",
+    feature = "self_remove_proposal"
+))]
+impl Debug for SelfRemoveProposal {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("SelfRemoveProposal").finish()
+    }
+}
```

Comment:
[nit] You can just derive Debug, we implement manually above only to use `pretty_bytes`

Reply:
Ah, makes sense! Thanks! Done.